commands.rst

```diff
@@ -0,0 +1,116 @@
+| :ref:`Upside-down mode <1b7b>`                   | 1B 7B    | ESC {   | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Set CPI mode <1bc1>`                       | 1B C1    | ESC Á   | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Select Codepage <1c7d26>`                  | 1C 7D 26 | FS } &  | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Select character size <1d21>`              | 1D 21    | GS !    | ✔        | ✔       |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Reverse print mode <1d42>`                 | 1D 42    | GS B    | ✔        | ✔       |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Select double strike mode <1b47>`          | 1B 47    | ESC G   |          | ✔       |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Right side character spacing <1b20>`       | 1B 20    | ESC SP  | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Line spacing <1b33>`                       | 1B 33    | ESC 3   | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Select 1/6 inch line spacing <1b32>`       | 1B 32    | ESC 2   | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Select 1/8 inch line spacing <1b30>`       | 1B 30    | ESC 0   | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Select justification <1b61>`               | 1B 61    | ESC a   | ✔        | ✔       |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Left margin <1d4c>`                        | 1D 4C    | GS L    | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Motion units <1d50>`                       | 1D 50    | GS P    | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Print area width <1d57>`                   | 1D 57    | GS W    | ✔        |         |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Select Cut Mode and Cut Paper <1d56>`      | 1D 56    | GS V    | ✔        | ✔       |
++--------------------------------------------------+----------+---------+----------+---------+
+| :ref:`Full Cut <1b6d>`                           | 1B 6D    | ESC m   |          | ✔       |
```

Comment:
Okay there was some confusion here, the phoenix commands spreadsheet have the full cut and partial cut commands backwards (according to the Epson ESC/POS docs). I had changed the docs here to represent what was in phoenix. I will swap them so that they honor the proper byte command.

Reply:
We deviate from the competitors implementation, they deviate from each, and even themselves sometimes in some places. As long as these commands reflect the correct behavior I'm fine with it. I think you effectively clarified the differences down in the implementation sections for these commands. 

Since this is a summary table, I think it makes sense to leave the partial cut for Reliance unchecked so there is no misinterpretation of the cutter capabilities.

Reply:
Very well, I will review both phoenix and reliance to ensure consistency across the commands.